annotation/consume_trigger.go

```diff
@@ -204,6 +204,8 @@ func (SliceAccessPrestring) String() string {
 // FldAccess is when a value flows to a point where a field of it is accessed, and so it must be non-nil
 type FldAccess struct {
 	ConsumeTriggerTautology
+	Field  *types.Var
+	Method *types.Func
```

Comment:
IIUC, one and only one of these two must be nonnil, right? Can we make that explicit in the comments here?

Alternatively, we can just store `Sel types.Object` here and use type assertions to determine whether it is `*types.Var` or `*types.Func`

Reply:
Good idea. Went ahead with the latter option.